.github/actions/build-nominatim/action.yml

```diff
@@ -6,7 +6,8 @@ runs:
     steps:
         - name: Install prerequisites
           run: |
-            sudo apt-get install -y -qq libboost-system-dev libboost-filesystem-dev libexpat1-dev zlib1g-dev libbz2-dev libpq-dev libproj-dev python3-psycopg2 python3-pyosmium python3-dotenv python3-psutil python3-jinja2
+            sudo apt-get install -y -qq libboost-system-dev libboost-filesystem-dev libexpat1-dev zlib1g-dev libbz2-dev libpq-dev libproj-dev libicu-dev python3-psycopg2 python3-pyosmium python3-dotenv python3-psutil python3-jinja2
+            sudo pip install PyICU
```

Comment:
I only find the python-pyicu package on debian and on xenial and bionic distributions of ubuntu. It isn't available on my ubuntu distribution (focal) so I went for the PyICU python package. Is pip making any trouble?

Reply:
Ah, just for the fun of it the Python3 version is called `python3-icu`.

Using pip is fine. We just try to give installable packages the preference when they are available because they are less likely to cause clashes with other installed software.